op-deployer/pkg/deployer/pipeline/espresso.go

```diff
@@ -0,0 +1,53 @@
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/opcm"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/state"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func DeployEspresso(env *Env, intent *state.Intent, st *state.State, chainID common.Hash) error {
+	lgr := env.Logger.New("stage", "deploy-espresso")
+
+	chainIntent, err := intent.Chain(chainID)
+	if err != nil {
+		return fmt.Errorf("failed to get chain intent: %w", err)
+	}
+
+	chainState, err := st.Chain(chainID)
+	if err != nil {
+		return fmt.Errorf("failed to get chain state: %w", err)
+	}
+
+	if !chainIntent.EspressoEnabled {
+		lgr.Info("espresso batch inbox contract deployment not needed")
+		return nil
+	}
+
+	lgr.Info("deploying espresso contracts")
+	var nvo opcm.DeployNitroVerifierOutput
+	nvo, err = opcm.DeployNitroVerifier(env.L1ScriptHost, opcm.DeployNitroVerifierInput{
+		// TODO: get real PCR0
```

Comment:
When do we need to address this?

Reply:
Would be nice to have before the audit, definitely must-have for any deployments we'll want to make.